crates/storage/src/lazy/mapping.rs

```diff
@@ -38,12 +38,21 @@ use ink_primitives::Key;
 
 /// A mapping of key-value pairs directly into contract storage.
 #[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
-#[derive(Default)]
 pub struct Mapping<K, V> {
     offset_key: Key,
     _marker: PhantomData<fn() -> (K, V)>,
 }
 
+/// We implement this manually because the derived implementation adds trait bounds.
```

Comment:
What additional trait bounds are added here? 

I checked the `cargo-expand`-ed contracts using both versions and there was no difference between them.

Reply:
The `Default` derive requires `Default` for each type parameter. But for this `Mapping` we don't want these bounds. In substrate we have a `DefaultNoBounds` derive for this.